src/libstd/Cargo.toml

```diff
@@ -17,13 +17,13 @@ alloc = { path = "../liballoc" }
 panic_unwind = { path = "../libpanic_unwind", optional = true }
 panic_abort = { path = "../libpanic_abort" }
 core = { path = "../libcore" }
-libc = { path = "../rustc/libc_shim" }
-compiler_builtins = { path = "../rustc/compiler_builtins_shim" }
+libc = { version = "0.2", default-features = false, features = ['rustc-dep-of-std'] }
+compiler_builtins = { version = "0.1.0" }
 profiler_builtins = { path = "../libprofiler_builtins", optional = true }
 unwind = { path = "../libunwind" }
 
 [dev-dependencies]
-rand = "0.5"
+#rand = "0.5"
```

Comment:
I suspect this is because uncommenting rand here causes problems down the line?

Reply:
Ah sorry forgot to add a comment about this. This is a temporary holdover until https://github.com/rust-random/rand/pull/647 is merged, published, and integerated here.